### DISCUSSION TITLE
LightGBMTuner optimisation metric and custom metrics

### DISCUSSION THREAD

**Discussion by RashidBakirov** [optuna/optuna#5681] [Q&A]
While studying LightGBMTuner documentation, I couldn't find some details I was looking for: 
1) What is the validation set, the results on which is used as an optimisation target. Looking at this example https://github.com/optuna/optuna-examples/blob/main/lightgbm/lightgbm_tuner_simple.py , a list of datasets is passed in the argument `valid_sets` - does it mean that the optimisation metric is calculated on a combination of both of these?
2) Is it possible to include custom evaluation metric (for information, not the optimisation)? E.g. my dataset uses log-transform on the target; while I am happy to optimise RMSE on the transformed data, I'd also like to know what is the RMSE on the original data. I guess if this is not possible to achieve with LightGBMTuner, I'd have to go back to the "naive method" for optimisation.
